docs/concurent-jobs-with-configmaps.md

```diff
@@ -0,0 +1,77 @@
+# Concurrent Jobs With Configmaps Workload
+
+The Concurrent Jobs with Configmaps test playbook is `workloads/concurrent-jobs-with-configmaps.yaml`
+This workload test is designed to check how many of ConfigMaps and pods can slowdown the cluster.
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/concurrent-jobs-with-configmaps.yml or define Environment vars (See below)
+$ time ansible-playbook -vv -i inventory workloads/concurrent-jobs-with-configmaps.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`  
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`  
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`  
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`  
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `false`  
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `false`  
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `true`  
+Enables/disables running the workload pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`  
+Location of kubeconfig on orchestration host.
+
+### PBENCH_INSTRUMENTATION
+Default: `false`  
+Enables/disables running the workload wrapped by pbench-user-benchmark. When enabled, pbench agents can then be enabled (`ENABLE_PBENCH_AGENTS`) for further instrumentation data and pbench-copy-results can be enabled (`ENABLE_PBENCH_COPY`) to export captured data for further analysis.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`  
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the tooling playbook.
+
+### ENABLE_PBENCH_COPY
+Default: `false`  
+Enables/disables the copying of pbench data to a remote results server for further analysis.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`  
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`  
+Location of the ssh public key to authenticate to the pbench results server.
+
+### PBENCH_SERVER
+Default: There is no public default.  
+DNS address of the pbench results server.
+
+### NUMBER_OF_CONCURRENT_JOBS
+Default: 300
+Number of concurrent jobs with configmaps to create during workload.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`  
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
```

Comment:
Can we add the smoke testing vars similar to https://github.com/openshift-scale/workloads/blob/master/docs/nodevertical.md#smoke-test-variables please? This will be useful for smoke testing the workload.

Reply:
Done.
Added to file with variables and documentation.